android/example/facebook_messenger.js

```diff
@@ -0,0 +1,43 @@
+exports.window = function(value){
```

Comment:
Why is this file added? The messenger API's are not supported on Android, yet. Probably merge conflicts?